frontend/app/work_packages/controllers/details-tab-overview-controller.js

```diff
@@ -137,9 +137,49 @@ module.exports = function($scope,
   (function setupWorkPackageProperties() {
     var otherAttributes = WorkPackagesOverviewService.getGroupAttributesForGroupedAttributes('other', $scope.groupedAttributes);
 
-    angular.forEach($scope.workPackage.props.customProperties, function(customProperty) {
-      this.push(customProperty);
-    }, otherAttributes);
+    function getFormat(workPackage, prop, propName) {
+      return workPackage.props[propName] ?
+              workPackage.props[propName].format :
+              prop.type.toLowerCase();
+    }
+
+    function getValue(workPackage, prop, propName) {
+      if (workPackage.props[propName]) {
+        return workPackage.props[propName].raw;
```

Comment:
For most custom field types you want to directly access `workPackage.props[propName]`, e.g. `String`, `Integer`, `Float`, etc.

It is only `Formattable` that has a `raw` attribute. Like with description the meaning of the subproperties is:

* `format`: The format used to transform from raw to html (not that interesting for most clients I guess)
* `raw`: That is the string to display in an **editing** interface and send when editing... It will be transformed into html according to the format
* `html`: An already prepared html string, should be ready to use out of the box

**tl; dr:**
This line:
1. only works for custom fields of type `Formattable` ("Long Text" in the web interface), not for `Integer` et al
2. you should probably use the `html` subelement instead of `raw`, because otherwise line breaks are not rendered correctly